### DISCUSSION TITLE
Add Playfield Scaling

### DISCUSSION THREAD

**Discussion by philhk** [ppy/osu#28661] [Ideas] [answered]
Hey everyone,

I have an idea that could really help with gameplay on larger areas: adding an option to scale the playfield.

When playing on a large area it can be really hard to play some DT maps. While you can shrink your play area to make it easier this forces you to readjust your muscle memory which can be quite annoying.

My suggestion is to implement a feature that allows you to scale down the playfield. This means the visuals remain the same but are just smaller allowing you to keep your muscle memory intact while playing.

What do you think? Would this be useful?

**Comment by Doksha** [comment, marked as answer]
Search "scale" in options

**Comment by philhk** [comment]
Thank you so much for pointing that out. I feel a bit embarrassed for missing this in the settings.
